From d5cd39462de1dd289b00be106945af44e6224c29 Mon Sep 17 00:00:00 2001
From: Madara273 <ravenhoxs@gmail.com>
Date: Fri, 28 Feb 2025 04:29:27 +0000
Subject: [PATCH 13/15] net-tcp: Enable PLB by default

Signed-off-by: Madara273 <ravenhoxs@gmail.com>
---
 net/ipv4/tcp_ipv4.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/net/ipv4/tcp_ipv4.c b/net/ipv4/tcp_ipv4.c
index 7befa6a413094..d22a684e65799 100644
--- a/net/ipv4/tcp_ipv4.c
+++ b/net/ipv4/tcp_ipv4.c
@@ -2887,7 +2887,7 @@ static int __net_init tcp_sk_init(struct net *net)
 	atomic_set(&net->ipv4.tfo_active_disable_times, 0);
 
 	/* Set default values for PLB */
-	net->ipv4.sysctl_tcp_plb_enabled = 0; /* Disabled by default */
+	net->ipv4.sysctl_tcp_plb_enabled = 1; /* Disabled by default */
 	net->ipv4.sysctl_tcp_plb_idle_rehash_rounds = 3;
 	net->ipv4.sysctl_tcp_plb_rehash_rounds = 12;
 	net->ipv4.sysctl_tcp_plb_suspend_rto_sec = 60;
-- 
2.47.2

