From 1f5c7b3faa5a6f36a24f8428e269e43c9918b836 Mon Sep 17 00:00:00 2001
From: Madara273 <ravenhoxs@gmail.com>
Date: Sat, 22 Feb 2025 19:36:57 +0000
Subject: [PATCH 02/10] MPTCP -- Build mptcp

Signed-off-by: Madara273 <ravenhoxs@gmail.com>
---
 net/Kconfig  | 1 +
 net/Makefile | 1 +
 2 files changed, 2 insertions(+)

diff --git a/net/Kconfig b/net/Kconfig
index 630d7f05651f..cfc232b741f2 100644
--- a/net/Kconfig
+++ b/net/Kconfig
@@ -95,6 +95,7 @@ source "net/wireguard/Kconfig"
 source "net/ipv4/Kconfig"
 source "net/ipv6/Kconfig"
 source "net/netlabel/Kconfig"
+source "net/mptcp/Kconfig"
 
 endif # if INET
 
diff --git a/net/Makefile b/net/Makefile
index 09155b7ffb1f..62b71713c2ed 100644
--- a/net/Makefile
+++ b/net/Makefile
@@ -21,6 +21,7 @@ obj-$(CONFIG_TLS)		+= tls/
 obj-$(CONFIG_XFRM)		+= xfrm/
 obj-$(CONFIG_UNIX_SCM)		+= unix/
 obj-$(CONFIG_NET)		+= ipv6/
+obj-$(CONFIG_MPTCP)		+= mptcp/
 obj-$(CONFIG_BPFILTER)		+= bpfilter/
 obj-$(CONFIG_PACKET)		+= packet/
 obj-$(CONFIG_NET_KEY)		+= key/
-- 
2.47.2

